include/BSMPT/bounce_solution/action_calculation.h

```diff
@@ -0,0 +1,818 @@
+// Copyright (C) 2024 Lisa Biermann, Margarete Mühlleitner, Rui Santos, João
+// Viana
+//
+// SPDX-FileCopyrightText: 2024 Lisa Biermann, Margarete Mühlleitner, Rui
+// Santos, João Viana
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#pragma once
+
+/**
+ * @file
+ */
+
+// Remove warning from <experimental/filesystem>
+#define _LIBCPP_NO_EXPERIMENTAL_DEPRECATION_WARNING_FILESYSTEM
```

Comment:
Why use experimental filesystem at all?
We can just upgrade to c++17 and use the non experimental filesystem

Reply:
This was used when we saved the path iterations. Removed. We did not upgrade to c++17 to not break backwards compatibility.

Reply:
I don't think we break backwards compatibility. We already have 17 in use in some cases, so we should use it everywhere. I will make a seperate PR for this